airbyte-tests/build.gradle

```diff
@@ -61,12 +61,6 @@ task acceptanceTests(type: Test) {
         exceptionFormat "full"
     }
     mustRunAfter test
-
-    dependsOn ':airbyte-integrations:connectors:source-postgres:airbyteDocker'
```

Comment:
these should not have been here anymore. we use pinned versions of postgres source and destination